src/main/java/seedu/address/logic/commands/ListTodayCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
```

Comment:
I think `seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS` will be deprecated soon based on what WJ said? Should switch to the new variant that was created in [the base commit](https://github.com/AY2324S1-CS2103T-T15-2/tp/commit/d98620f4051eef0dfbae5920d3441caf2d3a287a).

Reply:
yepz, will be `seedu.address.model.ModelPrescription.PREDICATE_SHOW_ALL_PRESCRIPTIONS`;